px-kpi-list.html

```diff
@@ -50,26 +52,26 @@
 <dom-module id="px-kpi-list">
   <template>
     <style include="px-kpi-styles"></style>
-
-
-    <div class="flex flex--row flex--middle u-mv--">
-      <div class="flex flex--left flex__item">
-        <span class="caps epsilon secondary u-mv0">{{label}}</span>
-      </div>
-      <div class="flex flex--right flex__item">
-        <px-icon id="statusIcon" icon$="{{statusIcon}}" class="u-mr--" style$="color:{{statusColor}};"></px-icon>
-        <span class="epsilon">{{statusLabel}}</span>
-      </div>
-    </div>
+		<template is="dom-if" if="{{_checkIsHeaderDataPresent()}}">
+			<div class="flex flex--row flex--middle u-mv--">
+			  <div class="flex flex--left flex__item">
+				<span class="caps epsilon secondary u-mv0">{{label}}</span>
+			  </div>
+			  <div class="flex flex--right flex__item">
+				<px-icon id="statusIcon" icon$="{{statusIcon}}" class="u-mr--" style$="color:{{statusColor}};"></px-icon>
+				<span class="epsilon">{{statusLabel}}</span>
+			  </div>
+			</div>
+		</template>
     <ul id="kpiList" class="list-ui u-mb--">
       <template is="dom-repeat" items="{{values}}">
         <li class="list-ui__item flex flex--row flex--middle u-ph0 u-pv-">
           <div class="flex flex--left flex__item">
             <span id="listLabel">{{item.label}}</span>
           </div>
           <div class="flex flex--right flex--bottom flex__item">
-            <span class="delta">{{item.value}}</span>
-            <span class="caps u-mb-- u-ml--">{{item.uom}}</span>
+            <span id="kpiListValue">{{item.value}}</span>
```

Comment:
Don't use IDs for non-unique elements, this can cause issues (this is a dom-repeat element, so this element can be rendered multiple times). Use a class instead.